FindTown/FindTown/Presentation/TownIntroScene/TownIntro/TownIntroViewController.swift

```diff
@@ -0,0 +1,243 @@
+//
+//  TownIntroViewController.swift
+//  FindTown
+//
+//  Created by 장선영 on 2023/01/29.
+//
+
+import UIKit
+import FindTownCore
+import FindTownUI
+import RxSwift
+import RxCocoa
+
+/// 동네소개 페이지
+final class TownIntroViewController: BaseViewController, UIScrollViewDelegate {
+    
+    // MARK: - Properties
+    
+    var viewModel: TownIntroViewModel?
+    
+    // MARK: - Views
+    
+    /// 소개
+    private let townIntroView = UIView()
+    private let townIntroTitleLabel = FindTownLabel(text: "어떤 동네인가요?", font: .subtitle4)
+    private let townIntroLabel = FindTownLabel(text: "", font: .body4, textColor: .grey6, textAlignment: .left)
+    
+    /// 동네 분위기
+    private let townMoodView = UIView()
+    private let townMoodTitleLabel = FindTownLabel(text: "동네 분위기", font: .subtitle4)
+    private let townMoodCollectionView = TownMoodCollectionView()
+    
+    /// 교통
+    private let trafficView = UIView()
+    private let trafficTitleLabel = FindTownLabel(text: "교통", font: .subtitle4)
+    private let trafficNearLabel = FindTownLabel(text: "주변 지하철 노선", font: .body4)
+    private let trafficTipStackView = UIStackView()
+    
+    /// 숫자로 보는 동네
+    private let townRankView = UIView()
+    private let townRankTitleLabel = FindTownLabel(text: "숫자로 보는 동네", font: .subtitle4)
+    private let townRankInfoButton = UIButton()
+    private let townRankScrollView = UIScrollView()
+    private let townRankStackView = UIStackView()
+    private let townRankInfoImageView = UIImageView()
+    
+    /// 근처 핫플레이스
+    private let hotPlaceView = UIView()
+    private let hotPlaceTitleLabel = FindTownLabel(text: "근처 핫플레이스", font: .subtitle4)
+    private let hotPlaceCollectionView = HotPlaceCollectionView()
+    
+    // MARK: - Life Cycle
+    
+    init(viewModel: TownIntroViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Functions
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func setupView() {
+        self.view.backgroundColor = FindTownColor.grey1.color
```

Comment:
```suggestion
    override func setupView() {
        self.stackView.backgroundColor = FindTownColor.grey1.color
```

전체 view의 backgroundcolor를 세팅을 하셔서 scroll 했을 때 여백까지 grey로 보여요..! 이렇게 하시면 원하시는 화면이 나오는 것 같습니다..! :)

Reply:
앗 그렇네요 감사합니다 !!